scripts/backup_outputs.py

```
@@ -0,0 +1,42 @@
+current_dir = Path(os.getcwd())
+parent_dir = Path(os.path.realpath(__file__)).parent.parent
+
+if current_dir != parent_dir:
+    raise Exception(f"Script can only be run in {parent_dir}")
+
+def backup_outputs():
+    assets_dir = "assets"
+    output_base = "outputs"
+
+    # Generate timestamp only once per run
+    timestamp_str = datetime.now().strftime("%Y%m%d_%H%M%S")
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)